app/models/appointment.rb

```diff
@@ -54,6 +54,13 @@ def self.overdue_by(number_of_days)
     overdue.where('scheduled_date <= ?', Date.current - number_of_days.days)
   end
 
+  def self.eligible_for_reminders(days_overdue)
```

Comment:
Can we use a named argument? The plain method signature `Appointment.eligible_for_reminders(3)` looks strange.